lib/phonegap/cordova-dependence.js

```diff
@@ -69,9 +72,25 @@ module.exports.exec = function(projectDir, extEvents) {
                 var cordovaCommand = 'npm install cordova --save';
                 extEvents.emit('log', 'Adding Cordova dependency with: "' + cordovaCommand + '"');
                 extEvents.emit('log', 'This may take a few minutes ...');
+
+                var spinner = ora('installing cordova').start();
+
+                var startTime = Date.now();
+                analytics.trackEvent([], 0, {
+                    short_message: "npm-install-cordova-start",
+                    _params: []
+                });
+
                 var child = shell.exec(cordovaCommand, {
                     silent: true
                 }, function(code, stdout, stderr) {
+                    spinner.stop();
+                    var dT = Date.now() - startTime;
+                    var dTsec = Math.round(dT / 1000) + "s"; // analytics does not like numbers,
+                    analytics.trackEvent([dTsec], code, {
+                        short_message: "npm-install-cordova-finish",
+                        _params: [dTsec]
```

Comment:
Absolutely! Yeah, your input was what I was looking for here.
Does it make sense to have a generic '_value' so we don't grow our columns with every event we choose to track?  We could foresee things like `_length, _count, _retries, _difference, _total ???` all being pushed into their own properties if we allowed it. Where we can infer the meaning of 'value' based on what the event message is.
Definitely agree _params is NOT the right place, just didn't know the data format was so flexible.